src/index.js

```diff
@@ -1,12 +1,13 @@
-const path = require('path');
 const express = require('express');
 const Server = require('./my-server');
 
 const app = express();
 
-app.get('/', (req, res) => {
-    res.sendFile(path.join(__dirname, '../index.html'));
-});
+app.use(express.static('static'));
+
+if (require.main === module)
+    (async () => await new Server(app).start())();
```

Comment:
It's nice that you tried to properly handle the promise returned by `Server.start`. But since module loading is synchronous, you still can't properly handle the result of your wrapper. Let's just create an instance and call the method to simplify the code.